### PR TITLE
ADT now works on all installables

### LINKED ISSUE #4248: Architect Deployment Test still doesn't work on Agendas
Related to #4174.

### PR DESCRIPTION
**Pull request by lostgeek**
Closes #4248

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
When will I learn how to read? Thanks for catching this.

File: test/clj/game_test/cards/agendas.clj [hunk 1]
```diff
@@ -130,11 +130,11 @@
       (is (= 3 (count (:discard (get-corp)))) "trashed")
       (is (= 1 (count-tags state)) "Runner took 0 tags"))))
 
-(deftest architect-deployment-test
-  ;; Accelerated Beta Test
+(deftest ^:test-refresh/focus architect-deployment-test
```
Comment (NoahTheDuke) [inline test/clj/game_test/cards/agendas.clj]: 👀

File: test/clj/game_test/cards/agendas.clj [hunk 2]
```diff
@@ -130,11 +130,11 @@
...
+  ;; Architect Deployment Test
```
Comment (NoahTheDuke) [inline test/clj/game_test/cards/agendas.clj]: Goodness, thank you for all these fixes.

File: test/clj/game_test/cards/agendas.clj (hunk 1 above)
Comment (lostgeek) [reply test/clj/game_test/cards/agendas.clj]: whoops

**Pull request merged**